runtime/scripts/question.js

```diff
@@ -416,6 +417,7 @@ Question.prototype = /** @lends Numbas.Question.prototype */
         this.xml.selectSingleNode('parts').appendChild(xml);
         var j = this.parts.length;
         var p = Numbas.createPartFromXML(xml_index, xml,'p'+j,this,null,this.store, scope);
+        this.events.trigger('createExtraPartFromXML', xml_index);
```

Comment:
I don't think this is necessary, since the `addExtraPart` event will be triggered immediately afterwards.